frontend/src/metabase/palette/shortcuts/global.ts

```diff
@@ -142,4 +142,24 @@ export const globalShortcuts = {
     shortcut: ["c e"],
     shortcutGroup: "global" as const,
   },
+
+  "toggle-dark-mode": {
+    get name() {
+      return t`Toggle dark mode`;
+    },
+
+    shortcut: ["$mod+Shift+KeyL"],
+    shortcutGroup: "global" as const,
```

Comment:
this does not work for me 😢

Reply:
If you have the 1 password chrome plugin, by default it swallows this shortcut. Also I hear Loom likes this shortcut as well